src/main/java/com/epam/ta/reportportal/core/log/impl/AsyncCreateLogHandler.java

```diff
@@ -60,27 +59,24 @@
 
 	@Override
 	@Nonnull
+	//TODO check saving an attachment of the item of the project A in the project's B directory
 	public EntryCreatedRS createLog(@Nonnull SaveLogRQ createLogRQ, MultipartFile file, ReportPortalUser.ProjectDetails projectDetails) {
 
 		TestItem testItem = testItemRepository.findById(createLogRQ.getTestItemId())
 				.orElseThrow(() -> new ReportPortalException(ErrorType.TEST_ITEM_NOT_FOUND, createLogRQ.getTestItemId()));
-
 		validate(testItem, createLogRQ);
 
 		Log log = new LogBuilder().addSaveLogRq(createLogRQ).addTestItem(testItem).get();
-		try {
-			logRepository.save(log);
-		} catch (Exception exc) {
-			throw new ReportPortalException("Error while Log instance creating.", exc);
-		}
+		logRepository.save(log);
+
 		if (null != file) {
```

Comment:
What if file not null, but content is empty?

Reply:
If file exists we should save it, cause it can be empty just to 'mark' any log